language/Portuguese_Brazil/strings.po

```diff
@@ -7296,28 +7343,28 @@ msgstr "Pressione %s pra apagar"
 #: plugins/inputmacro/inputmacro_menu.lua:533
 msgctxt "plugin-inputmacro"
 msgid "[no macros]"
-msgstr ""
+msgstr "[não há macros]"
 
 #: plugins/inputmacro/inputmacro_menu.lua:538
 msgctxt "plugin-inputmacro"
 msgid "Add macro"
-msgstr ""
+msgstr "Adicionar macro"
 
 #: plugins/timer/init.lua:89
 #, lua-format
 msgctxt "plugin-timer"
 msgid "%03d:%02d:%02d"
-msgstr ""
+msgstr "%03d:%02d:%02d"
 
 #: plugins/timer/init.lua:98
 msgctxt "plugin-timer"
 msgid "Current time"
-msgstr "Tempo Atual"
+msgstr "Tempo atual"
```

Comment:
This is the elapsed time for the current emulation session, not the current wall clock time.  Is this the most appropriate translation?

Reply:
Fixed, but the original string "Current time" MUST be fixed to avoid this confusion. ;)

Reply:
To an English speaker, the original string is understood to mean elapsed time for current session due to context.  I realise it isn’t always easy to infer context when looking at a large list of messages like this.

The `msgctx` for messages that have it does tell you what part of MAME the message comes from, and there are source locations as well.  I can’t realistically promise to add more notes for translators.  We don’t have a team dedicated to localisation – it basically falls to me to try and make sure it happens, and I’m already over-stretched.